app/helpers/application_helper.rb

```diff
@@ -1,2 +1,21 @@
 module ApplicationHelper
+  def nightly_rate_helper
```

Comment:
These are awesome! This is a great way to pull stuff out of the view and improve readability